manim/animation/transform.py

```diff
@@ -403,6 +532,17 @@ class ApplyMatrix(ApplyPointwiseFunction):
         The origin point for the transform. Defaults to ``ORIGIN``.
     kwargs
         Further keyword arguments that are passed to :class:`ApplyPointwiseFunction`.
+
+    Examples
+    --------
+
+    .. manim:: ApplyMatrix
+        :quality: low
+
+        class ApplyMatrix(Scene):
```

Comment:
Circular reference for `ApplyMatrix` (the scene and the animation have the same name). Causes the docs to fail.
```suggestion
    .. manim:: ApplyMatrixExample
        :quality: low

        class ApplyMatrixExample(Scene):
```